README.md

```diff
@@ -1,15 +1,25 @@
 # S.O.P.P. - Satellite Orbit Prediction Processor
 
-## Run
+SOPP is an open-source tool for calculating satellite interference to radio astronomy observations.
+
+![alt text](https://github.com/NSF-Swift/satellite-overhead/blob/documentation/FBD.png?raw=true)
+
+## Running SOPP
+
+### Install the requirements:
+```bash
+>> pip3 install -r requirements.txt
+```
 
 ### Setting up the supplemental files
 
 #### Active Satellites TLE File
 There should be a [TLE](https://en.wikipedia.org/wiki/Two-line_element_set) file, containing the satellites that you 
-want to search through, at `supplements/active_satellites.tle` under the root directory.
+want to search through, at `supplements/satellites.tle` under the root directory. If you do not specify a config
+file, SOPP will download a list of TLEs from Space-trak.org
 
 #### Config file
-There should be a file at `supplements/.config` under the root directory to set default values for a reservation. 
+There should be a file at `supplements/.config` under the root directory to set default observation values for a reservation. 
 The following is an example of a config file:
 
     [RESERVATION]
```

Comment:
Can @ellem95 add descriptions for each of these configuration keys?

Reply:
Added!